src/screens/ui/ActivityListScreen.tsx

```diff
@@ -12,21 +18,22 @@ import { Box, HorizontalCalendar } from '@shared/ui';
 type Props = BottomTabScreenProps<AppletDetailsParamList, 'ActivityList'>;
 
 const ActivityListScreen: FC<Props> = props => {
+  const appletId = props.route.params.appletId;
+
   const isFocused = useIsFocused();
 
+  useOnFocus(() => {
+    AnalyticsService.track(MixEvents.AppletView, {
+      [MixProperties.AppletId]: appletId,
+    });
+  });
+
   return (
     <Box flex={1}>
       <UploadRetryBanner accessibilityLabel="upload-banner" />
       <HorizontalCalendar mt={8} />
       <StreamingStatusBar appletId={props.route.params.appletId} mb={20} />
-
-      {isFocused && (
-        <ActivityGroups
-          flex={1}
-          px={14}
-          appletId={props.route.params.appletId}
-        />
-      )}
+      {isFocused && <ActivityGroups flex={1} px={14} appletId={appletId} />}
```

Comment:
Please make sure the groups don't disappear right away during the screen transition animation (e.g. open InProgressActivityScreen)

Reply:
FYI: https://reactnavigation.org/docs/use-focus-effect/#delaying-effect-until-transition-finishes

Reply:
They will and it was before